spate_190313.R

```diff
@@ -1,16 +1,6 @@
 library(DAutilities)
 library(decisionSupport)
 
-# Cory_changes
-#The file path should be the same as the .Rproj (or relative to it)
-#If Negusse
-#filepath<-"D:/PhD/Turkana/spate/crop_production/Single_intervention/"  
-#If Cory
-#This is the same as the working directory... getwd()
-#filepath<-"/Users/macbook/Dropbox/University of Bonn/PhD Projects/Negusse/02_Spate_irrigation/Spate/"
-#filepath<-"D:/Git/Spate"  
-#I removed all of these paths. As Cory said, this should be the project directory. Then we won't have to adjust this each time we run it on a new computer.
-
```

Comment:
Ok thanks! However, does it mean that the path will be easily detectable? If so, how about on the welfare decision part?will it work without the path?

Reply:
Hi Negusse, Eike and I both responded by email but it did not get posted here for some reason. 

Eike said 'Yes, as long as this is a real git project.

E'

and I said : 'The beauty of using RStudio is that if you call getwd() it is always the same as the directory where the .rproj is saved. 

Cory'